samples/ABNF/tranquil.abnf

```diff
@@ -0,0 +1,61 @@
+; Source:  https://github.com/fjolnir/Tranquil
+; License: BSD 3 Clause License
+; Modified for standard compatibility by René Schwaiger
```

Comment:
> This is the sort of in-the-wild discrepancy that's both natural and expected; and also part of the reason we need samples in the first place.

I do not think that it makes that much sense to add incorrect files. Otherwise we could just add the following “ABNF code” as (part of) a sample file 😀:

```
Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
deserunt mollit anim id est laborum.
```

The [ABNF TextMate grammar](https://github.com/sanssecours/ABNF.tmbundle/blob/master/Syntaxes/ABNF.tmLanguage) will also (correctly) mark parts of the original `tranquil.abnf` grammar as incorrect:

<img width="607" alt="tranquil" src="https://cloud.githubusercontent.com/assets/691989/19927057/58caa5f6-a0f8-11e6-91b9-0ab8fa4f43bc.png">